relay-pool.ts

```diff
@@ -60,6 +61,10 @@ export class ConnectionPool
         }
     }
 
+    setSigner(signer: Signer) {
+        this.signer = signer;
+    }
```

Comment:
why?

Reply:
Because this part requires a pool. But there is no Signer at this time.
https://github.com/BlowaterNostr/blowater/blob/c9b0db17d46bcfe4a9620ab8cc759dbb11a211a5/app/UI/app.tsx#L61-L68